mapswipe_workers/mapswipe_workers/utils/user_management.py

```diff
@@ -158,3 +158,52 @@ def permission_denied(request_object):
             return True
         else:
             raise HTTPError(e, request_object.text)
+
+
+def add_team(email, team_id):
+    """Add teamId attribute for user."""
+    try:
+        fb_db = firebaseDB()  # noqa E841
+
+        # check if team exist in firebase
+        if not fb_db.reference(f"v2/teams/{team_id}").get():
+            raise CustomError(f"can't find team in firebase: {team_id}")
+
+        # get user by email
+        try:
+            user = auth.get_user_by_email(email)
+        except auth.UserNotFoundError:
+            raise CustomError(f"can't find user in firebase: {email}")
+
+        # set team token in firebase
+        ref = fb_db.reference(f"v2/users/{user.uid}")
+        ref.update({"teamId": team_id})
+        logger.info(f"added teamId {team_id} for user {email} - {user.uid}.")
+
+    except Exception as e:
+        logger.info(f"could not add teamId attribute for user.")
+        raise CustomError(e)
+
+
+def remove_team(email):
+    """Remove teamId attribute for user."""
+    try:
+        fb_db = firebaseDB()  # noqa E841
+
+        # get user by email
+        try:
+            user = auth.get_user_by_email(email)
+        except auth.UserNotFoundError:
+            raise CustomError(f"can't find user in firebase: {email}")
+
+        # get teamId from firebase
+        team_id = fb_db.reference(f"v2/users/{user.uid}/teamId").get()
+
+        # remove teamId attribute for user in firebase
+        ref = fb_db.reference(f"v2/users/{user.uid}")
+        ref.update({"teamId": None})
```

Comment:
Does this result in the `teamId` attribute having a null value, or is it removed entirely from the db?

Reply:
basically this is the same, we set a null value in firebase and this results in the attribute being removed. It's just another way of deleting an attribute in firebase, but with update you could theoretically update several attributes at once.

Reply:
Ok, makes sense. I was mostly asking to understand how to approach this on the app side. So I can mostly assume that if `teamId` is present, it has a valid value (ie, not `null`).